packages/cloudflare/src/cli/build/bundle-server.ts

```diff
@@ -94,13 +94,12 @@ fetch = globalThis.fetch;
 const CustomRequest = class extends globalThis.Request {
   constructor(input, init) {
     if (init) {
-      init = {
-        ...init,
-        cache: undefined,
-        // https://github.com/cloudflare/workerd/issues/2746
-        // https://github.com/cloudflare/workerd/issues/3245
-        body: init.body instanceof __cf_stream.Readable ? ReadableStream.from(init.body) : init.body,
-      };
+      init.cache = undefined;
+      // https://github.com/cloudflare/workerd/issues/2746
+      // https://github.com/cloudflare/workerd/issues/3245
+      Object.defineProperty(init, "body", {
+        value: init.body instanceof __cf_stream.Readable ? ReadableStream.from(init.body) : init.body;
+      });
```

Comment:
Notes: 
- when debugging `{...init}` miss headers, method, ....
- `Object.defineProperty` is used because `init.body = ...;` would sometimes throw because the prop is not writeable

Reply:
Could you please add this comment as an actual code comment?

Reply:
The test would fail without that